images/svc-base/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM registry.scality.com/federation/nodesvc-base:7.10.6.0
+
+WORKDIR /usr/src/app
+
+# Keep the .git directory in order to properly report version
+COPY ./package.json yarn.lock ./
+
+ENV PYTHON=python3.9
+ENV PY_VERSION=3.9.7
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+    jq \
+    python \
+    git \
+    build-essential \
+    ssh \
+    ca-certificates \
+    wget \
+    libffi-dev \
+    zlib1g-dev \
```

Comment:
```suggestion
    zlib1g-dev \
    && apt-get clean \
```

Reply:
Added to both federation and default docker files